boards/arm/stm32wb5mm_dk/stm32wb5mm_dk.yaml

```diff
@@ -7,7 +7,7 @@ toolchain:
   - gnuarmemb
   - xtools
 ram: 256
-flash: 1024
+flash: 876
```

Comment:
Why it is reduced?

https://github.com/zephyrproject-rtos/zephyr/blob/36c12482c6ff4349307c93c67e9cd914ed0c6d1c/dts/arm/st/wb/stm32wb.dtsi#L143

and board [documentation](https://www.st.com/en/evaluation-tools/stm32wb5mm-dk.html) mention 1MB.

Reply:
@javad123javad IIUC, this change is not related to this patch or even to the series. If so, please take it out. Eventually this can be submitted separately in a dedicated PR.

As a rule of thumb, please don't include unrelated changes in a patch.